src/doc/gui_manual/Operators/OperatorTypes/ICS/LCS_operator.rst

```diff
@@ -0,0 +1,85 @@
+.. _LCS operator:
+
+Lagrangian Coherent Structure operator
```

Comment:
add `(LCS)` abbreviation here to as in `Lagrangian Coherent Structure (LCS) operator`